src/transform/convert/create-material-fields.spec.js

```diff
@@ -55,35 +55,32 @@ function callback({
   const debugResultHandling = debug.extend('resultHandling');
   const inputData = getFixture('input.json');
   const expectedResults = getFixture('output.json');
-  const results = testContext.default(inputData);
-
   try {
+    const results = testContext.default(inputData);
     debugResultHandling(JSON.stringify(results));
-    expect(results).to.eql(expectedResults);
-  } catch (err) {
-    errorHandling(err);
-  }
 
-  function errorHandling(err) {
+    if (results) {
+      expect(results).to.eql(expectedResults);
+      return;
+    }
```

Comment:
On real run, in case of error, the transformation has failed and ends there, thrown error is catch on lower level and added to blob metadata. In this case this "if" is just guard that on expected error test case does not expect success results.